config/locales/fr.yml

```diff
@@ -123,6 +123,9 @@ fr:
             active_initiatives: Pétitions actives
             see_all_initiatives: Voir toutes les pétitions
     meetings:
+      application_helper:
+        filter_scope_values:
+          all: All
```

Comment:
This french key isn't translated

Reply:
Nice catch @Quentinchampenois! Indeed the french translation "Tous" is missing.